TexasInstruments/SNx4HC595/GHIElectronics.TinyCLR.Drivers.TexasInstruments.SNx4HC595/SNx4HC595.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GHIElectronics.TinyCLR.Devices.Gpio;
+using GHIElectronics.TinyCLR.Devices.Spi;
+
+namespace GHIElectronics.TinyCLR.Drivers.TexasInstruments.SNx4HC595 {
+    public class SNx4HC595 {
+        private readonly GpioPin latch;
+        private readonly SpiDevice spiBus;
+
+        public SNx4HC595(string spiName, int chipSelect, int latchPin) {
+            this.spiBus = SpiController.FromName(spiName).GetDevice(
```

Comment:
Follow the pattern of the other drivers where the `GpioPin` and `SpiDevice` are passed in, expose a `GetConnectionSettings` method. See https://github.com/ghi-electronics/TinyCLR-Drivers/blob/dev/ShijiLighting/APA102C/APA102C.cs

Reply:
Done!